tests/integration/cloudformation/api/test_stacks.py

```diff
@@ -293,3 +294,160 @@ def stack_process_is_finished(cfn_client, stack_name):
         "PROGRESS"
         not in cfn_client.describe_stacks(StackName=stack_name)["Stacks"][0]["StackStatus"]
     )
+
+
+@pytest.mark.skip(reason="Not implemented")
+@pytest.mark.aws_verified
+def test_drift_detection_on_lambda(deploy_cfn_template, cfn_client, lambda_client, snapshot):
+    snapshot.add_transformer(snapshot.transform.cloudformation_api())
+    stack = deploy_cfn_template(
+        template_path=os.path.join(os.path.dirname(__file__), "../../templates/lambda_url.yaml")
+    )
+
+    lambda_client.update_function_configuration(
+        FunctionName=stack.outputs["LambdaName"],
+        Runtime="python3.8",
+        Description="different description",
+        Environment={"Variables": {"ENDPOINT_URL": "localhost.localstack.cloud"}},
+    )
+
+    drift_detection = cfn_client.detect_stack_resource_drift(
+        StackName=stack.stack_name, LogicalResourceId="Function76856677"
+    )
+
+    snapshot.match("drift_detection", drift_detection)
+
+
+@pytest.mark.aws_validate
+@pytest.mark.skip(reason="Not Implemented")
+def test_linting_error_during_creation(cfn_client):
+    stack_name = f"stack-{short_uid()}"
+    bad_template = {"Resources": "", "Outputs": ""}
+
+    with pytest.raises(botocore.exceptions.ClientError) as ex:
+        cfn_client.create_stack(StackName=stack_name, TemplateBody=json.dumps(bad_template))
+
+    error_response = ex.value.response
+
+    assert error_response["Error"]["Code"] == "ValidationError"
+    assert "Template format error" in error_response["Error"]["Message"]
```

Comment:
I'd encourage the use of a snapshot here. Might seem a bit unnecessary but asserts like this have failed us in the past unfortunately: `assert "Template format error" in error_response["Error"]["Message"]`